site/data/docs/custom-wallet-list.mdx

```diff
@@ -267,8 +256,9 @@ const connectors = connectorsForWallets([
   {
     groupName: 'Others',
     wallets: [
-      wallet.walletConnect({ chains }),
+      wallet.injected({ chains, shimDisconnect: true }),
```

Comment:
`shimDisconnect` is already default truthy.

```suggestion
      wallet.injected({ chains }),
```

Reply:
Maybe we can omit other usages while we are at it?

Reply:
Ah, nice catch. Done 👍